virtio-devices/src/vhost_user/vu_common_ctrl.rs

```diff
@@ -197,7 +198,7 @@ impl VhostUserHandle {
                 .set_vring_base(
                     queue_index,
                     queue
-                        .avail_index_from_memory(mem)
+                        .used_index_from_memory(mem)
```

Comment:
Question: why is this change needed?

Reply:
This is needed to keep the whole code in sync. In [here](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/virtio-devices/src/transport/pci_device.rs#L506-L511) we use it as well on the restore codepath.
The reason for using `used_index_from_memory()` instead of `avail_index_from_memory()` is because we want to start at the correct index based on what was done by the backend. Let's say the backend fully processed a descriptor right before the pause, it will have updated the index in the used ring. Relying on `used` is more correct than relying on `avail` since `used` is updated by the backend while `avail` is updated by the guest.